### PR TITLE
Create a generator that converts the json file into the README.md file

### PR DESCRIPTION
**Pull request by Andy-set-studio**
## About this PR
I've made a little generator with node that will convert the `checklist.json` file into `README.md`. It's bare-bones ES5 to make sure it's as easy to run as possible. The only prerequisite is that the user has node installed on their machine. 

I also split the readme up by creating `intro.md`. This is so we can continue to manage the content of it in a reasonable fashion.

To run the generator,  run: `node generator/index.js`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,26 +62,26 @@ Aims to be the **biggest** checklist of inclusive design considerations for the
...
+- [ ] Make content easier to find and improve search results with structured data [Resource #1](https://developers.google.com/search/docs/guides/prototype)
 - [ ] Use textual labels to make voice activation cues obvious
 - [ ] Do not mark up subheadings/straplines with separate heading elements
 - [ ] Ensure primary calls to action are easy to recognize and reach
 - [ ] Avoid images of text — text that cannot be translated, selected, or understood by assistive tech
 - [ ] Provide a print stylesheet (single column, with interactive content hidden)
 - [ ] Use well-established, therefore recognizable, icons and symbols
 - [ ] Subset fonts to just the characters needed
-- [ ] Instead of obstructing users with CAPTCHAs, use [honeypots](https://en.wikipedia.org/wiki/Honeypot_(computing))
+- [ ] Instead of obstructing users with CAPTCHAs, use [honeypots [Resource #1](https://en.wikipedia.org/wiki/Honeypot_(computing))
```
Comment (peter-mouland) [inline README.md:85]: seems to be an extra `[` here

File: README.md [hunk 2]
```diff
@@ -62,26 +62,26 @@ Aims to be the **biggest** checklist of inclusive design considerations for the
...
 - [ ] Begin long, multi-section documents with a table of contents
-- [ ] Don't make users perform actions to reveal content unless completely necessary
+- [ ] Don"t make users perform actions to reveal content unless completely necessary
```
Comment (peter-mouland) [inline README.md:87]: not sure `"` is necessary :)

**Review by peter-mouland — CHANGES_REQUESTED** [review]
Nice work! but looks like a couple some minor tweaks are needed (see comments)

**Comment by Andy-set-studio** [comment]
@peter-mouland, Yeh they're all from `checklist.json`, so that could do with a tidy up :)

**Comment by Heydon** [comment]
@hankchizljaw Not clear on the status of this pull request. Can I merge or are you still tweaking?

**Comment by Andy-set-studio** [comment]
Yeh the generator is good to go mate. I think we could tackle the pre-commit in a separate PR. 

I think we could work out how to integrate and the pre-commit as a group. I started off a bit of a discussion in #1. We should probably split out into another issue though?

**Comment by Heydon** [comment]
@hankchizljaw Sounds good (the new issue I mean)

**Comment by Andy-set-studio** [comment]
I’ve started a new discussion in #23. Thanks for merging the code in, @Heydon! 🙂

**Pull request merged**